tests/components/geo_json_events/test_geo_location.py

```diff
@@ -58,7 +58,9 @@ async def test_entity_lifecycle(
         (-31.0, 150.0),
         {ATTR_NAME: "Properties 1"},
     )
-    mock_entry_2 = _generate_mock_feed_entry("2345", "Title 2", 20.5, (-31.1, 150.1))
+    mock_entry_2 = _generate_mock_feed_entry(
```

Comment:
I don't think this really testing for the case that the PR is fixing. Don't we need a case where the name property is an int?

Reply:
https://github.com/home-assistant/core/blob/333769d06e949fa1ce60e1b3cd9da91a3fb79cd2/tests/components/geo_json_events/test_geo_location.py#L62

Both occurences of 271310188 are an int here on set-up of the entity, as might happen in the feed.

When we later check for it, both times we expect it to be a str.

https://github.com/home-assistant/core/blob/333769d06e949fa1ce60e1b3cd9da91a3fb79cd2/tests/components/geo_json_events/test_geo_location.py#L96

https://github.com/home-assistant/core/blob/333769d06e949fa1ce60e1b3cd9da91a3fb79cd2/tests/components/geo_json_events/test_geo_location.py#L101

Reply:
Ah I see, yes, I had glanced over the presence of the dict with `ATTR_NAME` for some reason. I think I'm a bit confused why there's type conversion going on in `_generate_mock_feed_entry()`, too (and just of the title). Couldn't you just pass in a string for the title and not need to convert there?

Reply:
The intent was to simulate for the same conversion as happens against the entity. If it's not required, we could remove it but I'm fairly sure there was errors.